### PR TITLE
Fix editing multi-sheets

### PR DESCRIPTION
**Pull request by bosschaert**
## Description

Editing existing multi-sheets (spreadsheets that contain more than 1 sheet) is currently broken. This fixes it.

Test URLs:
Before: https://da.live/sheet#/da-sites/da-status/bosschae/data/tags
After: https://multi-sheet--da-live--adobe.hlx.live/sheet#/da-sites/da-status/bosschae/data/tags

## Related Issue

Fixes: #182

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by bosschaert** [comment]
The PW tests are not very reliable when run in the GH action. They are when run from my machine, but this needs to be looked at which I will do separately.

**Comment by auniverseaway** [comment]
What are your reproduction steps. I show multi-sheet working as expected:

https://da.live/sheet#/da-sites/xsc-wknd-commerce/configs

File: blocks/sheet/index.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ async function getData(url) {
   const names = json[':names'];
 
   // Single sheet
-  if (json.data) {
+  if (json[':type'] === 'sheet') {
```
Comment (bstopp) [inline blocks/sheet/index.js:54]: There a test for this?

Comment (bosschaert) [reply blocks/sheet/index.js:54]: There are no unit tests yet for any of this area, but I'll add a test for this change.

**Comment by bosschaert** [comment]
> What are your reproduction steps. I show multi-sheet working as expected:
> 
> https://da.live/sheet#/da-sites/xsc-wknd-commerce/configs

See here: https://da.live/sheet#/da-sites/da-status/bosschae/data/tags

The JSON can be seen here:
https://main--da-status--da-sites.hlx.page/bosschae/data/tags.json

**Comment by bosschaert** [comment]
Hi @bstopp I've added unit tests for the code changed now.

**Review by chrischrischris — APPROVED** [review]

**Pull request merged**